documentation/docs/use/commands/pmm-admin.md

```diff
@@ -1,25 +1,27 @@
 # pmm-admin - PMM Administration Tool
 
-## NAME
+`pmm-admin` is a command-line tool for administering PMM using a set of COMMAND keywords and associated FLAGS.
 
-`pmm-admin` - Administer PMM
+PMM communicates with the PMM Server via a PMM agent process.
```

Comment:
it communicates with PMM Server directly, it gets info about PMM Server from PMM Agent. I'm not sure if it should be here.